src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/CacheEntry.java

```diff
@@ -44,4 +44,6 @@
 
     void abortSubscribers();
 
-}
\ No newline at end of file
+    boolean expectSubscribers();
```

Comment:
Personally I think the name hasSubscribers() better reflects the intention of this method, but I agree neither are perfect!